akka-actor-testkit-typed/src/main/mima-filters/2.6.2.backwards.excludes/28473-upgrade-ScalaTest-to-3-1-0.excludes

```diff
@@ -0,0 +1,72 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit.assertionsHelper")
```

Comment:
I think the file name is supposed to be latest released, i.e. `2.6.1.backwards.excludes`

Reply:
done